client/src/app/categories/routes.js

```diff
@@ -1,11 +1,26 @@
 /**
- * Group components for code splitting in Web pack
- *
+ * Components are lazy-loaded
  * http://router.vuejs.org/en/advanced/lazy-loading.html
+ *
+ * An alternative syntax, to group all components into a single
+ * Webpack 'chunk', or file is as follows:
+ *
+ * const Categories = r => require.ensure([], () => r(require('./main.vue')), 'categories-bundle')
+ * const Form = r => require.ensure([], () => r(require('./form.vue')), 'categories-bundle')
+ *
  */
 
-const Categories = r => require.ensure([], () => r(require('./main.vue')), 'categories-bundle')
-const Form = r => require.ensure([], () => r(require('./form.vue')), 'categories-bundle')
+const Categories = (resolve) => {
+    require.ensure(['./main'], () => {
+      resolve(require('./main')) // eslint-disable-line global-require
```

Comment:
Ah - I thought you wanted it reverted, with comments. Have just pushed an update.